### PR TITLE
CTL9-8-Add BottomNavigation

### PR DESCRIPTION
**Pull request by gordanailic**

### CODE REVIEW

File: app/src/main/java/com/example/cocktails/CocktailsFragment.kt [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+
+
+class CocktailsFragment : Fragment() {
+
+
+    override fun onCreate(savedInstanceState: Bundle?) {
```
Comment (a-radovanovic-levi9-com) [inline app/src/main/java/com/example/cocktails/CocktailsFragment.kt]: For now we don't need onCreate Method.

File: app/src/main/java/com/example/cocktails/FavoritesFragment.kt [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+
+
+class FavoritesFragment : Fragment() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
```
Comment (a-radovanovic-levi9-com) [inline app/src/main/java/com/example/cocktails/FavoritesFragment.kt]: For now we don't need onCreate Method.

File: app/src/main/java/com/example/cocktails/ProfileFragment.kt [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+
+class ProfileFragment : Fragment() {
+
+
+    override fun onCreate(savedInstanceState: Bundle?) {
```
Comment (a-radovanovic-levi9-com) [inline app/src/main/java/com/example/cocktails/ProfileFragment.kt]: For now we don't need onCreate Method.

File: app/src/main/java/com/example/cocktails/CocktailsFragment.kt [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+        super.onCreate(savedInstanceState)
+
+    }
+
+    override fun onCreateView(
```
Comment (a-radovanovic-levi9-com) [inline app/src/main/java/com/example/cocktails/CocktailsFragment.kt]: We can replace onCreateView with class Cocktails : Fragment(R.layout.fragment_cocktails) {}

File: app/src/main/java/com/example/cocktails/FavoritesFragment.kt [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+        super.onCreate(savedInstanceState)
+    }
+
+    override fun onCreateView(
```
Comment (a-radovanovic-levi9-com) [inline app/src/main/java/com/example/cocktails/FavoritesFragment.kt]: We can replace onCreateView with class Cocktails : Fragment(R.layout.fragment_cocktails) {}

File: app/src/main/java/com/example/cocktails/ProfileFragment.kt [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+        super.onCreate(savedInstanceState)
+
+    }
+
+    override fun onCreateView(
```
Comment (a-radovanovic-levi9-com) [inline app/src/main/java/com/example/cocktails/ProfileFragment.kt]: We can replace onCreateView with class Cocktails : Fragment(R.layout.fragment_cocktails) {}

**Review by a-radovanovic-levi9-com — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/style.xml [hunk 7]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <style name="BottomNavigation.ActiveItemTextAppearance" parent="TextAppearance.AppCompat">
+        <item name="android:textSize">22dp</item>
+        <item name="android:textStyle">italic</item>
+        <item name="android:fontFamily">@font/caveat</item>
+    </style>
+
+    <style name="ToolbarTitleTextAppearance">
+        <item name="android:textSize">22dp</item>
```
Comment (Levi9Daniel) [inline app/src/main/res/values/style.xml]: We should use SP as a unit for Text Size

File: app/src/main/res/values/style.xml [hunk 8]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <style name="BottomNavigation.ActiveItemTextAppearance" parent="TextAppearance.AppCompat">
+        <item name="android:textSize">22dp</item>
```
Comment (Levi9Daniel) [inline app/src/main/res/values/style.xml]: We should use SP as a unit for Text Size

File: app/src/main/res/values/strings.xml [hunk 9]
```diff
@@ -1,20 +1,7 @@
...
-    <string name="error_fragment">Error Fragment</string>
-    <string name="personal_settings">Personal Settings</string>
-    <string name="watch_trailer_1">Watch trailer</string>
-    <string name="watch_trailer_2">FREE</string>
-    <string name="rent_1">Rent By Day</string>
-    <string name="rent_2">From $1.99</string>
-    <string name="buy_1">Buy and Own</string>
-    <string name="buy_2">AT $9.99</string>
-    <string name="movie">Movie</string>
+    <string name="coctails">Coctails</string>
```
Comment (Levi9Daniel) [inline app/src/main/res/values/strings.xml]: Typo in both key and value..Should be Cocktails

File: app/src/main/res/navigation/my_nav.xml [hunk 10]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<navigation xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/my_nav"
+    app:startDestination="@id/cocktailsFragment">
+
+    <fragment
```
Comment (Levi9Daniel) [inline app/src/main/res/navigation/my_nav.xml:8]: Even though Android Studio does not warn us about this, we should set all the labels from the strings.xml file

File: app/src/main/res/layout/fragment_profile.xml [hunk 11]
```diff
@@ -0,0 +1,19 @@
...
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".ProfileFragment">
+
+    <TextView
+        android:id="@+id/textView2"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Profile"
```
Comment (Levi9Daniel) [inline app/src/main/res/layout/fragment_profile.xml]: Please use string resource file

File: app/src/main/res/layout/fragment_favorites.xml [hunk 12]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".FavoritesFragment">
+
+    <TextView
+        android:id="@+id/textView3"
```
Comment (Levi9Daniel) [inline app/src/main/res/layout/fragment_favorites.xml:10]: Please use string resource file. Also try not to use the fixed TextView height

File: app/src/main/res/layout/fragment_cocktails.xml [hunk 13]
```diff
@@ -0,0 +1,18 @@
...
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".CocktailsFragment">
+
+    <TextView
+        android:id="@+id/textView"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Coctail"
```
Comment (Levi9Daniel) [inline app/src/main/res/layout/fragment_cocktails.xml]: Please use string resource file

File: app/src/main/res/layout/activity_splash_screen.xml [hunk 14]
```diff
@@ -5,5 +5,44 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     tools:context=".SplashScreenActivity">
+    <!-- Dodajte Toolbar ovde -->
```
Comment (Levi9Daniel) [inline app/src/main/res/layout/activity_splash_screen.xml]: We should remove the unnecessary code comments

**Review by Levi9Daniel — CHANGES_REQUESTED** [review]

**Review by Levi9Daniel — APPROVED** [review]

**Pull request merged**